.github/workflows/docker.yml

```diff
@@ -20,7 +23,15 @@ jobs:
         with:
           distribution: corretto
           java-version: 19
+      
+      - name: Login to Docker
+        run: docker login -u $DOCKER_USERNAME -p $DOCKER_HUB
+
+      - name: Get Commit Time
+        run: echo "COMMIT_TIME=$(git show -s --format=%ci ${{ github.sha }})" >> $GITHUB_ENV`s
+
+      - name: Build and push Docker image for file-backup-local
+        run: gradle jib --image="$DOCKER_USERNAME/file-backup-local:${{ env.COMMIT_TIME }}"
```

Comment:
I think this can be `./gradlew :file-backup-local:jibDockerBuild` and it gets the tag etc. from the config in `build.gradle`.

I'll do some testing...